### DISCUSSION TITLE
Allow for custom path to .NET SDK just like you can with Android SDK.

### DISCUSSION THREAD

**Discussion by SpicerXD** [godotengine/godot-proposals#8831] [Scripting]
After tooling around with Android and then .NET support.
I noticed you couldn't specify a custom path for the .NET SDK, like you can with almost any other external tooling in Godot.
Like signtool, rcedit, Android SDK, etc.

I like to work in portable development environments, which Godot does WONDERFULLY. And I imagine many others like to as well.
And this was the only external toolset I had to uses scripts to make portable for developing with in Godot.

Just some food for thought if anyone braver than me at writing and testing C++ code sees this. XD

**Comment by Calinou** [comment]
Thanks for the proposal! Consolidating in https://github.com/godotengine/godot-proposals/issues/1941.
